### ISSUE TITLE
Error: Cannot read property 'getPeripheral' of undefined

### ISSUE THREAD

**Issue by koktaildotcom** [athombv/apps-sdk-issues#71]
Think there is missing an is null check or the cache is cold at this point.
Scenario is in branch: https://github.com/koktaildotcom/homey-mi-flora/tree/feature/master-cannot-read-from-null
```
{ Error: Cannot read property 'getPeripheral' of undefined
    at Object.JSON.parse (/opt/homey-client/system/helpers/jsonfns.js:1:1190)
    at JSON.parse (<anonymous>)
    at jsonToObject (/opt/homey-client/system/helpers/jsonfns.js:1:1083)
    at function.e.data.map.e (/opt/homey-client/system/manager/ManagerApps/bootstrap/sdk/v2/lib/HomeyClient.js:1:1379)
    at Array.map (<anonymous>)
    at HomeyClient._onMessage (/opt/homey-client/system/manager/ManagerApps/bootstrap/sdk/v2/lib/HomeyClient.js:1:1372)
    at emitTwo (events.js:126:13)
    at process.emit (events.js:214:7)
    at emit (internal/child_process.js:762:12)
    at _combinedTickCallback (internal/process/next_tick.js:142:11)
  message: 'Cannot read property \'getPeripheral\' of undefined' }
```

**Comment by WeeJeWel** [comment]
Which Homey version?

**Comment by koktaildotcom** [comment]
Homey v2.0.5-rc.2. I'll try a PTP before submitting more issues.

**Comment by WeeJeWel** [comment]
Fixed in v2.0.5-rc.3
